### ISSUE TITLE
Network issue, not fully initialized or service availability prevent module initilization

### ISSUE THREAD

**Issue by morpheus65535** [nidhaloff/deep-translator#138]
This part of code:
https://github.com/nidhaloff/deep-translator/blob/0cfe8e507c46b4e21e458fe0d4d2195dd28eab59/deep_translator/constants.py#L202-L208

Raise an unhandled exception when api.cognitive.microsofttranslator.com is unreachable:
```
Mar 09 16:26:13 HTPC bazarr.htpc[667]: Traceback (most recent call last):
Mar 09 16:26:13 HTPC bazarr.htpc[667]:   File "/opt/bazarr/bazarr/main.py", line 20, in <module>
Mar 09 16:26:13 HTPC bazarr.htpc[667]:     from init import *  # noqa E402
Mar 09 16:26:13 HTPC bazarr.htpc[667]:   File "/opt/bazarr/bazarr/init.py", line 211, in <module>
Mar 09 16:26:13 HTPC bazarr.htpc[667]:     init_binaries()
Mar 09 16:26:13 HTPC bazarr.htpc[667]:   File "/opt/bazarr/bazarr/init.py", line 192, in init_binaries
Mar 09 16:26:13 HTPC bazarr.htpc[667]:     from utils import get_binary
Mar 09 16:26:13 HTPC bazarr.htpc[667]:   File "/opt/bazarr/bazarr/utils.py", line 27, in <module>
Mar 09 16:26:13 HTPC bazarr.htpc[667]:     from deep_translator import GoogleTranslator
Mar 09 16:26:13 HTPC bazarr.htpc[667]:   File "/opt/bazarr/bazarr/../libs/deep_translator/__init__.py", line 3, in <module>
Mar 09 16:26:13 HTPC bazarr.htpc[667]:     from .google_trans import GoogleTranslator
Mar 09 16:26:13 HTPC bazarr.htpc[667]:   File "/opt/bazarr/bazarr/../libs/deep_translator/google_trans.py", line 5, in <module>
Mar 09 16:26:13 HTPC bazarr.htpc[667]:     from .constants import BASE_URLS, GOOGLE_LANGUAGES_TO_CODES, GOOGLE_LANGUAGES_SECONDARY_NAMES
Mar 09 16:26:13 HTPC bazarr.htpc[667]:   File "/opt/bazarr/bazarr/../libs/deep_translator/constants.py", line 205, in <module>
Mar 09 16:26:13 HTPC bazarr.htpc[667]:     microsoft_languages_response = requests.get(microsoft_languages_api_url)
Mar 09 16:26:13 HTPC bazarr.htpc[667]:   File "/opt/bazarr/bazarr/../libs/requests/api.py", line 75, in get
Mar 09 16:26:13 HTPC bazarr.htpc[667]:     return request('get', url, params=params, **kwargs)
Mar 09 16:26:13 HTPC bazarr.htpc[667]:   File "/opt/bazarr/bazarr/../libs/requests/api.py", line 61, in request
Mar 09 16:26:13 HTPC bazarr.htpc[667]:     return session.request(method=method, url=url, **kwargs)
Mar 09 16:26:13 HTPC bazarr.htpc[667]:   File "/opt/bazarr/bazarr/../libs/requests/sessions.py", line 529, in request
Mar 09 16:26:13 HTPC bazarr.htpc[667]:     resp = self.send(prep, **send_kwargs)
Mar 09 16:26:13 HTPC bazarr.htpc[667]:   File "/opt/bazarr/bazarr/../libs/requests/sessions.py", line 645, in send
Mar 09 16:26:13 HTPC bazarr.htpc[667]:     r = adapter.send(request, **kwargs)
Mar 09 16:26:13 HTPC bazarr.htpc[667]:   File "/opt/bazarr/bazarr/../libs/requests/adapters.py", line 519, in send
Mar 09 16:26:13 HTPC bazarr.htpc[667]:     raise ConnectionError(e, request=request)
Mar 09 16:26:13 HTPC bazarr.htpc[667]: requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.cognitive.microsofttranslator.com', port=443): Max retries exceeded with url: /languages?api-version=3.0&scope=translation (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fad68e34310>: Failed to establish a new connection: [Errno -2] Name or service not known'))
```

**Comment by nidhaloff** [comment]
Thanks for opening the issue. I moved this part of the code to the MicrosoftTranslator class. This should be fixed in v1.8.0 

Feel free to re-open the issue otherwise.

**Comment by morpheus65535** [comment]
Appreciated, thanks!
